### PR TITLE
Wireless tags platform

### PR DESCRIPTION
**Pull request by sergeymaysak**
**Description:**

Documentation for Wireless Tags platform. Includes sensors, binary sensors and switches.

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.wirelesstag.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+The following metrics can be controlled:
+
+* (`temperature`): Control arm/disarm temperature monitoring.
+* (`humidity`): Control arm/disarm humidity monitoring.
+* (`motion`): Control arm/disarm motion and door open/close events monitoring.
+* (`light`): Control monitoring of light changes.
+* (`moisture`): Control monitoring of water level/soil moisture for water sensor.
+
+Arm/Disam of motion switch is required to receive motion and door binary sensors events.
```
Comment (dale3h) [inline source/_components/switch.wirelesstag.markdown]: - Typo in "Disarm"

File: source/_components/sensor.wirelesstag.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{% configuration %}
+  monitored_conditions:
+    description: The metrics types to monitor; valid values are specified below
+    required: true
+    type: list
+{% endconfiguration %}
+
+The following metrics can be monitored:
+
+* (`temperature`): Value is in Celcius or in Fahrenheit (accourding to your settings at Tag Manager).
```
Comment (dale3h) [inline source/_components/sensor.wirelesstag.markdown]: - Typo in "Celsius"

**Review by dale3h — CHANGES_REQUESTED** [review]
A few typos, but other than that it looks good. Now let's see if we can figure out why it's failing.

**Review by dale3h — DISMISSED** [review]

File: source/_components/binary_sensor.wirelesstag.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+description: "Instructions on how to integrate Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:49
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Binary Sensor
+ha_iot_class: "Local Push"
+---
```
Comment (frenck) [inline source/_components/binary_sensor.wirelesstag.markdown]: `ha_version` is missing.

File: source/_components/sensor.wirelesstag.markdown [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+description: "Instructions on how to integrate your Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:50
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
+---
```
Comment (frenck) [inline source/_components/sensor.wirelesstag.markdown]: `ha_version` is missing.

File: source/_components/switch.wirelesstag.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+description: "Instructions on how to integrate your Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:52
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Switch
+ha_iot_class: "Local Push"
+---
```
Comment (frenck) [inline source/_components/switch.wirelesstag.markdown]: `ha_version` is missing.

File: source/_components/wirelesstag.markdown [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+description: "Instructions on how to integrate your Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:32
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Hub
+ha_iot_class: "Local Push"
+---
```
Comment (frenck) [inline source/_components/wirelesstag.markdown]: `ha_version` is missing.

File: source/_components/binary_sensor.wirelesstag.markdown [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  monitored_conditions:
+    description: The conditions types to monitor; valid values are specified below
+    required: true
+    type: list
+{% endconfiguration %}
+
+The following conditions can be monitored:
+
+* (`presence`): On means in range, Off means out of range.
+* (`motion`): On when when movement was detected, Off when clear.
```
Comment (frenck) [inline source/_components/binary_sensor.wirelesstag.markdown]: `On when when movement was detected` -> `On when a movement was detected`

File: source/_components/binary_sensor.wirelesstag.markdown [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+* (`door`): On when when door is open, Off when door is closed.
```
Comment (frenck) [inline source/_components/binary_sensor.wirelesstag.markdown]: `On when when door is open, Off when door is closed.` -> `On when a door is open, Off when the door is closed.`

File: source/_components/switch.wirelesstag.markdown [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+The following metrics can be controlled:
+
+* (`temperature`): Control arm/disarm temperature monitoring.
+* (`humidity`): Control arm/disarm humidity monitoring.
+* (`motion`): Control arm/disarm motion and door open/close events monitoring.
+* (`light`): Control monitoring of light changes.
+* (`moisture`): Control monitoring of water level/soil moisture for water sensor.
+
+Arm/Disarm of motion switch is required to receive motion and door binary sensors events.
+Others are needed only if you want to receive push notifications from tags on specific range of changes for temperature, humidity, light or moisture.
```
Comment (frenck) [inline source/_components/switch.wirelesstag.markdown]: - `on specific range`  -> `on a specific range`
- `changes for temperature`  -> `changes in temperature`

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks, @sergeymaysak.
Pretty large and great contribution 🏆 

I've reviewed the docs and found small things. Could you please take a look? 👍

File: source/_components/switch.wirelesstag.markdown (hunk 5 above)
Comment (sergeymaysak) [reply source/_components/switch.wirelesstag.markdown]: @frenck thanks! what version would you like me to put here? as I'm not aware to what version  this contribution will go finally ;) so i left no version intentionally...

File: source/_components/wirelesstag.markdown (hunk 6 above)
Comment (sergeymaysak) [reply source/_components/wirelesstag.markdown]: @frenck thanks!  same here - what version would you like me to put here? as I'm not aware to what version  this contribution will go finally ;) so i left no version intentionally...

File: source/_components/binary_sensor.wirelesstag.markdown (hunk 7 above)
Comment (sergeymaysak) [reply source/_components/binary_sensor.wirelesstag.markdown]: @frenck thanks! will correct shortly...

File: source/_components/switch.wirelesstag.markdown (hunk 5 above)
Comment (frenck) [reply source/_components/switch.wirelesstag.markdown]: General recommendation is to pick the next "major"  release, which is `0.67`.

Comment (sergeymaysak) [reply source/_components/switch.wirelesstag.markdown]: @frenck got it, thx! in progress with it...

**Comment by sergeymaysak** [comment]
@frenck I've just committed requested changes and general note on main WirelessTags hub page regarding local push notification. Please have a look. Thank you )

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
@sergeymaysak, Approved ✅ 
Made some more minor tweaks, I hope you don't mind.


Can be merged as soon as the parent PR gets merged. 😃

**Comment by sergeymaysak** [comment]
@frenck, sorry for bothering, but based on testing of my PR by volunteers, it has been found that Tag Manager does not support `https`, so I've added `warning` on this. Additionally, I've added notion of Cloud Polling usage for each component. May I ask you to review/approve these changes?

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for the notify @sergeymaysak.
Re-approved 👍

**Comment by frenck** [comment]
Merged and tagged for milestone `0.72`, since this version has already been tagged `rc`.

Thanks, @sergeymaysak! Great first-time contribution m8!

bitmoji

**Pull request merged**